.vscode/settings.json

```diff
@@ -4,5 +4,6 @@
     "**/.pnp.*": true
   },
   "typescript.tsdk": "implementations/typescript/.yarn/sdks/typescript/lib",
-  "typescript.enablePromptUseWorkspaceTsdk": true
+  "typescript.enablePromptUseWorkspaceTsdk": true,
+  "java.configuration.updateBuildConfiguration": "interactive"
```

Comment:
This line is an unrelated change, please remove it from this commit.

Reply:
taking it out , but it does break the json file

Reply:
Hmm, interesting does vscode break for you without this change?

Reply:
The json breaking may be because the previous line has an extra comma now, it should be removed too

Reply:
Silly vscode inserted the java config and i didnt realise it had changed. Removed now :)